contents/blog/product-engineer-vs-software-engineer.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2022-12-01
+title: "Product engineer vs software engineer: what's the difference?"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+categories: ["Startups", "Guides"]
+---
+
+There are always arguments about what is the best way to run a tech company, how you should structure a company, and what roles you should have. Tech companies are endlessly trying to innovate in company composition to get work done effectively.
+
+A recent addition to the debate is the role of the product engineer. The role of the product engineer changed the way companies think about engineering. Is a greater focus on product necessary to succeed now? Are older structures and engineering roles outdated?
+
+To help you think through this yourself, we’ll compare the role of product engineer with the role of software engineer.
```

Comment:
```suggestion
Software companies are changing. Where once product managers and software engineers dominated, now a new role is emerging: the [product engineer](/blog/what-is-a-product-engineer).

Want proof? Look at this graph:

![product-engineer-trends](../images/blog/product-engineer-trend.png)

Online searches for product engineers have grown over 80% in the last year and are 3x greater now than just two years ago.

Product engineers are changing the way companies think about engineering. To help explain why, we’ll compare the role of product engineer with the traditional (and still very necessary) role of full-stack software engineer.
```

Added this search trend graph for some social proof – I'm wondering if we could back this up with some data from a recruiting website? Someone must have done some research in this area?

Minor SEO note: It's always a good idea to link to high value pages higher in the copy, not just the footer. Google gives more weight to links higher up the page.

Reply:
I've tried to find some data from a recruiting site, but haven't had any luck.